### PR TITLE
CLIENT-SPECIFICATION: add lowercasing of commands

### PR DESCRIPTION
**Pull request by MasterOdin**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

In preparation of landing tldr-lint@0.0.10 and resolving #5085 and #5255, I figured it would be good to add to the client specification about the lowercasing of file names as a transparent step to end users.

### CODE REVIEW

**Comment by bl-ue** [comment]
Won't we have to send out notifications to all of the clients about this change @sbrl?

File: CLIENT-SPECIFICATION.md
Comment (bl-ue) [inline CLIENT-SPECIFICATION.md:74]: Is this definitely correct?  Both version sound correct to me and I can't make up my mind.  I'm slightly inclined towards 'mixed'.
```suggestion
Page names MAY contain mixed capitalization - such page names MUST be transparently lowercased. For example, the page name:
```

**Comment by bl-ue** [comment]
The with this addition, the section becomes a bit unstructured. https://github.com/tldr-pages/tldr/blob/MasterOdin-patch-1/CLIENT-SPECIFICATION.md#page-names

Is there any way to fix it?

File: CLIENT-SPECIFICATION.md
Comment (owenvoke) [reply CLIENT-SPECIFICATION.md:84]: Whoops, didn't mean to "request changes" this was merely a comment.

Comment (bl-ue) [reply CLIENT-SPECIFICATION.md:84]: Maybe we can do that with the previous one, too.

Comment (MasterOdin) [reply CLIENT-SPECIFICATION.md]: Agreed. I have reworked it so both lines / examples follow this minimal format, keeping the one code block below that gives example invocations of tldr.

**Review by owenvoke — APPROVED** [review]

File: CLIENT-SPECIFICATION.md
Comment (bl-ue) [inline CLIENT-SPECIFICATION.md:62]: The dashse don't read well to me.  What about this?
```suggestion
Page names MAY contain spaces (e.g. `git status`), and such page names MUST be transparently concatenated with dashes (`-`). For example, the page name `git checkout` becomes `git-checkout`. Page names MAY contain mixed capitalization, and such page names MUST be transparently lowercased. For example, the page name `eyeD3` becomes `eyed3`.
```

Comment (MasterOdin) [reply CLIENT-SPECIFICATION.md]: done

File: CLIENT-SPECIFICATION.md
Comment (bl-ue) [inline CLIENT-SPECIFICATION.md:62]: One last thing.  There are two separate concepts here, and a newline between them is helpful.

```suggestion
Page names MAY contain spaces (e.g. `git status`), and such page names MUST be transparently concatenated with dashes (`-`). For example, the page name `git checkout` becomes `git-checkout`.

Page names MAY contain mixed capitalization, and such page names MUST be transparently lowercased. For example, the page name `eyeD3` becomes `eyed3`.
```

**Review by bl-ue — APPROVED** [review]

**Review by schneiderl — APPROVED** [review]
Looks nice to me.

**Comment by schneiderl** [comment]
Merging this as we have 3 approvals already. If there is any other concern in regards to this change, please reopen the discussion.

**Comment by owenvoke** [comment]
This probably could have done with a version bump to v1.5 before merging.

**Comment by bl-ue** [comment]
@owenvoke: #5428

**Pull request merged**
